src/app/content/components/Page/highlightManager.ts

```diff
@@ -76,10 +79,29 @@ const onSelectHighlight = (
   services.setPendingHighlight(highlight);
 });
 
-const createHighlighter = (services: Omit<HighlightManagerServices, 'highlighter'>) => {
+const createHighlighter = (services: Omit<HighlightManagerServices, 'highlighter'>, intl: IntlShape) => {
 
   const highlighter: Highlighter = new Highlighter(services.container, {
-    onClick: (highlight) => onClickHighlight({ ...services, highlighter }, highlight),
+    formatMessage: intl.formatMessage,
+    onClick: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
+    onFocusIn: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
+    // Without defer when user focus highlight with TAB and the click on the card the activeElement
+    // will be set to a <body> element for some reason
+    onFocusOut: () => defer(() => {
+      // Do not clear focus from highlight if it was moved to the Card component or to another highlight
+      // We still want to clear focused highlight if user TAB outside of it, for example to figure link.
+      const activeElement = assertDocument().activeElement;
+      if (
+        activeElement
+        && findFirstAncestorOrSelf(
+          activeElement,
+          (node) => node.hasAttribute('data-highlight-card') || node.hasAttribute('data-for-screenreaders')
+        )
+      ) {
+        return;
+      }
+      services.getProp().clearFocus();
```

Comment:
the part about not clearing if the focus was moved to another highlight is supposed to have been done already because of this https://github.com/openstax/rex-web/pull/1107/files#diff-69f29777fa0e95134d5f3990ffbc23328b6078e8f0fbaf3d54faf3219e56941bR51

can you also move this whole `onFocusOut` handler up next to `onFocusHighlight`?

Reply:
We could remove `|| node.hasAttribute('data-for-screenreaders')` from this function and it will work however in this case it will first dispatch `clear` and then it will `focus` new highlight
with this check it will only `focus` new highlight, without clearing the old one before
this shouldn't change anything for the user but it is stopping unnecessary `dispatch`

I've moved the handler to separate function